pom.xml

```diff
@@ -99,7 +99,13 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>git</artifactId>
-            <version>2.4.0</version>
+            <version>2.4.0</version> <!-- should really be 2.6.4 to match scm-api but ok as git wass forward compat -->
+        </dependency>
+
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>scm-api</artifactId>
+            <version>2.0.3-SNAPSHOT</version>
```

Comment:
@KostyaSha you already have the dependency via git 2.4.0 (but just an older version) and we need the `SCMEvent.originOf(req)` helper method